build.gradle

```diff
@@ -53,10 +53,7 @@ allprojects {
 }
 
 dependencies {
-  implementation ("com.sonatype.nexus:nexus-platform-api:$nexusPlatformApiVersion") {
-    exclude group: 'org.eclipse.jgit', module: 'org.eclipse.jgit'
-  }
-  implementation 'org.eclipse.jgit:org.eclipse.jgit:5.13.3.202401111512-r'
+  implementation "com.sonatype.nexus:nexus-platform-api:$nexusPlatformApiVesion"
```

Comment:
No longer need to exclude `jgit` and explicitly define the version. Version pulled in through `nexus-platform` 5.0.0.-01 is safe.